cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/BuiltInRolesAdministrationTest.scala

```diff
@@ -0,0 +1,311 @@
+        p(
+          """
+            |To restore the role to its original capabilities two steps are needed: First, if not already done, execute `DROP ROLE architect`.
+            |Secondly, the following queries need to be executed:""".stripMargin)
+        query("CREATE ROLE architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT ACCESS ON DATABASE * TO architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT MATCH {*} ON GRAPH * TO architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 2)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT WRITE ON GRAPH * TO architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 2)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT NAME MANAGEMENT ON DATABASE * TO architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT INDEX MANAGEMENT ON DATABASE * TO architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT CONSTRAINT MANAGEMENT ON DATABASE * TO architect", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        p("The resulting `architect` role has the same privileges as the " +
+          "<<administration-built-in-roles-architect-privileges,original built-in `architect` role>>.")
+
+//        query("SHOW ROLE architect PRIVILEGES", ResultAssertions(p => true)) {
+//          resultTable()
+//        }
+      }
+    }
+    section("The `admin` role", "administration-built-in-roles-admin", "enterprise-edition") {
+      p("The `admin` can do the same as  the <<administration-built-in-roles-architect,`architect`>>, as well as manage databases, users, roles and privileges.")
+      section("Privileges of the `admin` role", "administration-built-in-roles-admin-privileges", "enterprise-edition") {
+        query("SHOW ROLE admin PRIVILEGES",ResultAssertions(p => true)) {
+          resultTable()
+        }
+      }
+      section("How to recreate the `admin` role", "administration-built-in-roles-admin-recreate", "enterprise-edition") {
+        initQueries(
+          "DROP ROLE admin")  // setup so that later when the grant query gets executed, it will show systemUpdates: 1
+        note(p("In Neo4j 4.1.0 it is *not* possible to fully recreate a dropped admin role. " +
+          "Specifically, the ability to run procedures with the `@Admin` annotation like e.g. `dbms.listConfig` can *not* be restored." +
+          " Because of that, it is highly discouraged to drop the admin role."))
```

Comment:
```suggestion
          "Specifically, the ability to run procedures with the `@Admin` annotation, such as  `dbms.listConfig`, can *not* be restored." +
          " Because of that, dropping the `admin` role is strongly discouraged."))
```